### PR TITLE
Add a note to SHOW ALIASES as we now filter the database column

### PR DESCRIPTION
**Pull request by mnd999**
The database column is now filtered in the same was as SHOW DATABASES, so we add a note to the documentation to explain that.

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
Looks like you've updated the documentation!

Check out your changes at https://neo4j-docs-cypher-653.surge.sh

**Review by Hunterness — APPROVED** [review]

File: modules/ROOT/pages/administration/aliases.adoc [hunk 1]
```diff
@@ -283,6 +283,18 @@ The required privileges are described xref::administration/access-control/dbms-a
...
 | database
 | The name of the target database. label:default-output[]
+
+This column is filtered according to the `ACCESS` privileges of the user.
+However, some privileges enable users to see additional databases regardless of their `ACCESS` privileges:
+
+* Users with `CREATE/DROP/ALTER DATABASE` or `SET DATABASE ACCESS` privileges can see all standard databases.
+* Users with `CREATE/DROP COMPOSITE DATABASE` or `COMPOSITE DATABASE MANAGEMENT` privileges can see all composite databases.
+* Users with `DATABASE MANAGEMENT` privilege can see all databases.
+* Users always have visibility of the `system` database.
```
Comment (JPryce-Aklundh) [inline modules/ROOT/pages/administration/aliases.adoc]:
```suggestion
* Users can always see the `system` database.
```

File: modules/ROOT/pages/administration/aliases.adoc [hunk 2]
```diff
@@ -283,6 +283,18 @@ The required privileges are described xref::administration/access-control/dbms-a
 
 | database
 | The name of the target database. label:default-output[]
+
+This column is filtered according to the `ACCESS` privileges of the user.
+However, some privileges enable users to see additional databases regardless of their `ACCESS` privileges:
+
+* Users with `CREATE/DROP/ALTER DATABASE` or `SET DATABASE ACCESS` privileges can see all standard databases.
```
Comment (JPryce-Aklundh) [inline modules/ROOT/pages/administration/aliases.adoc]:
```suggestion
* Users with the `CREATE DATABASE`, `DROP DATABASE`, `ALTER DATABASE`, or `SET DATABASE ACCESS` privileges can see all standard databases.
```

File: modules/ROOT/pages/administration/aliases.adoc [hunk 3]
```diff
@@ -283,6 +283,18 @@ The required privileges are described xref::administration/access-control/dbms-a
...
+* Users with `CREATE/DROP COMPOSITE DATABASE` or `COMPOSITE DATABASE MANAGEMENT` privileges can see all composite databases.
```
Comment (JPryce-Aklundh) [inline modules/ROOT/pages/administration/aliases.adoc]:
```suggestion
* Users with the `CREATE COMPOSITE DATABASE`, `DROP COMPOSITE DATABASE`, or `COMPOSITE DATABASE MANAGEMENT` privileges can see all composite databases.
```

File: modules/ROOT/pages/administration/aliases.adoc [hunk 4]
```diff
@@ -283,6 +283,18 @@ The required privileges are described xref::administration/access-control/dbms-a
...
+* Users with `DATABASE MANAGEMENT` privilege can see all databases.
```
Comment (JPryce-Aklundh) [inline modules/ROOT/pages/administration/aliases.adoc]:
```suggestion
* Users with the `DATABASE MANAGEMENT` privilege can see all databases.
```

File: modules/ROOT/pages/administration/aliases.adoc [hunk 5]
```diff
@@ -283,6 +283,18 @@ The required privileges are described xref::administration/access-control/dbms-a
...
+* Users always have visibility of the `system` database.
+
+If a user has not been granted `ACCESS` privilege nor any of the above special cases above to the target database, then it is not visible to the user
```
Comment (JPryce-Aklundh) [inline modules/ROOT/pages/administration/aliases.adoc]:
```suggestion
If a user has not been granted any of these privileges to the target database, then it is not visible and this column will be `null`. 
```

File: modules/ROOT/pages/administration/aliases.adoc [hunk 6]
```diff
@@ -283,6 +283,18 @@ The required privileges are described xref::administration/access-control/dbms-a
...
+and this column will be `null`.
```
Comment (JPryce-Aklundh) [inline modules/ROOT/pages/administration/aliases.adoc:296]:
```suggestion
```

File: modules/ROOT/pages/administration/aliases.adoc (hunk 2 above)
Comment (Hunterness) [reply modules/ROOT/pages/administration/aliases.adoc]: If we change this here we should probably change it in the database chapter as well as it has the same text

File: modules/ROOT/pages/administration/aliases.adoc (hunk 5 above)
Comment (mnd999) [reply modules/ROOT/pages/administration/aliases.adoc]: This changes the meaning somewhat. The above list of privileges are not database specific, they appy to the DBMS. And we need to mention the ACCESS privilege really as that can apply to specific databases and it is not mentioned in the list above.

File: modules/ROOT/pages/administration/aliases.adoc [hunk 7]
```diff
@@ -283,6 +283,18 @@ The required privileges are described xref::administration/access-control/dbms-a
...
+
+This column is filtered according to the `ACCESS` privileges of the user.
+However, some privileges enable users to see additional databases regardless of their `ACCESS` privileges:
+
+* Users with the `CREATE DATABASE`, `DROP DATABASE`, `ALTER DATABASE`, or `SET DATABASE ACCESS` privileges can see all standard databases.
+* Users with the `CREATE COMPOSITE DATABASE`, `DROP COMPOSITE DATABASE`, or `COMPOSITE DATABASE MANAGEMENT` privileges can see all composite databases.
+* Users with the `DATABASE MANAGEMENT` privilege can see all databases.
+* Users can always see the `system` database.
+
+If a user has not been granted `ACCESS` privilege nor any of the above special cases above to the target database, then it is not visible to the user
```
Comment (mnd999) [inline modules/ROOT/pages/administration/aliases.adoc]: How about this?
```suggestion
If a user has not been granted the `ACCESS` privilege to the target database and none of the above special cases apply, then it is not visible to the user
```

File: modules/ROOT/pages/administration/aliases.adoc (hunk 5 above)
Comment (JPryce-Aklundh) [reply modules/ROOT/pages/administration/aliases.adoc]: Ok - I will delete the second "above" in "above special cases above" and then commit.

File: modules/ROOT/pages/administration/aliases.adoc (hunk 7 above)
Comment (JPryce-Aklundh) [reply modules/ROOT/pages/administration/aliases.adoc]: Great 👍

**Review by JPryce-Aklundh — APPROVED** [review]

**Pull request merged**